darwin/DarwinProcess.c

```diff
@@ -272,9 +272,16 @@ void DarwinProcess_setFromKInfoProc(Process* proc, const struct kinfo_proc* ps,
       proc->tpgid = ps->kp_eproc.e_tpgid;
       proc->tgid = proc->pid;
       proc->st_uid = ps->kp_eproc.e_ucred.cr_uid;
-      /* e_tdev = (major << 24) | (minor & 0xffffff) */
-      /* e_tdev == -1 for "no device" */
-      proc->tty_nr = ps->kp_eproc.e_tdev & 0xff; /* TODO tty_nr is unsigned */
+      {
```

Comment:
do we need the extra scoping/braces here?

Reply:
Not strictly needed, but IMHO they make things more readable. Otherwise you could reorder this after the translated flag and separate it by some blank line.

Reply:
Dropped the extra blocks.